projects/daq2/common/daq2_bd.tcl

```diff
@@ -1,33 +1,32 @@
 
 source $ad_hdl_dir/library/jesd204/scripts/jesd204.tcl
+source $ad_hdl_dir/projects/common/xilinx/data_offload_bd.tcl
```

Comment:
I think this can't be included in the common part since it will break Intel projects.